test/conformance/kuttl/verifyImages/clusterpolicy/standard/noconfigmap-diffimage-success/README.md

```diff
@@ -0,0 +1,4 @@
+# Title
```

Comment:
Please use the README template given in the [scaffolding directory](https://github.com/kyverno/kyverno/tree/main/test/conformance/kuttl/aaa_template_resources/scaffold).

Reply:
fixed